workflows/rules/common.smk

```diff
@@ -38,31 +38,36 @@ METADATA_DIR = hf.get_metadata_dir()
 # https://github.com/cytomining/pycytominer
 
 # Level 2 data: converted into parquet format
-CYTOTABLE_OUTPUT_DATA = hf.parquet_output()
-CYTOTABLE_OUTPUT_DATA_EXTENDED = expand(CYTOTABLE_OUTPUT_DATA, file_name=plate_name)
+CYTOTABLE_CONVERTED_PLATE_DATA = datapaths.build_path(
+    data_type="plate_data", use_converted=True
+)
+CYTOTABLE_CONVERTED_PLATE_DATA_EXTENDED = expand(
```

Comment:
What does `EXTENDED` mean in this context? Consider providing a comment here to describe this.

Reply:
Documentation added!